src/operations/estimated_document_count.ts

```diff
@@ -66,12 +40,12 @@ export class EstimatedDocumentCountOperation extends CommandOperation<number> {
     }
 
     super.executeCommand(server, session, cmd, (err, response) => {
-      if (err) {
+      if (err && (err as MongoServerError).code !== 26) {
```

Comment:
Should `(err as MongoServerError).code !== 26` exist anymore?  This logic was used with the `$collStats`-based version of `estimatedDocumentCount`.  Since we're no longer using the `$collStats`-based version, I don't think we should keep this logic.

Reply:
Removed.